docs: Fix minor text issues in connector/deltalake.rst (#26241)
prestodb/presto: presto-docs/src/main/sphinx/connector/deltalake.rst

=============================================== ========================================================= ============

Delta Lake connector reuses many of the modules existing in Hive connector.
Modules for connectivity and security such as S3, Azure Data Lake, and Glue metastore.

> The configurations for these modules is the same as those available in the :doc:`/connector/hive`.

Reviewer: How about rephrasing it something like this? 

```suggestion
Configuration options for these modules are identical to those described in the :doc:`/connector/hive`.
```
